Src/FluentAssertions/Primitives/DateTimeAssertions.cs

```diff
@@ -928,6 +928,36 @@ public AndConstraint<TAssertions> BeIn(DateTimeKind expectedKind, string because
         return new AndConstraint<TAssertions>((TAssertions)this);
     }
 
+    /// <summary>
+    /// Asserts that the <see cref="DateTime"/> does not represent a value in the <paramref name="unexpectedKind"/>.
+    /// </summary>
+    /// <param name="unexpectedKind">
+    /// The unexpected <see cref="DateTimeKind"/> that the current value must not represent.
+    /// </param>
+    /// <param name="because">
+    /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+    /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+    /// </param>
+    /// <param name="becauseArgs">
+    /// Zero or more objects to format using the placeholders in <paramref name="because" />.
+    /// </param>
+    public AndConstraint<TAssertions> NotBeIn(DateTimeKind unexpectedKind, string because = "", params object[] becauseArgs)
+    {
+        Execute.Assertion
+            .BecauseOf(because, becauseArgs)
+            .WithExpectation("Did not expect {context:the date and time} to be in " + unexpectedKind + "{reason}")
+            .Given(() => Subject)
```

Comment:
🔧 You only need a `Given` if you want to apply some kind of transformation.

Reply:
This was a suggestion from me to handle when the `Subject` is null and `NotBeIn` is wrapped in an `AssertionScope` - see the added test `Date_is_null_on_kind_check`.

I guess the alternative to (mis)using `Given` would be to split up the assertion chain and add a `if (success)` guard?

Reply:
Ah yes, now I see what you are doing. Not enough coffee on Saturday morning ;-)